lighthouse-core/test/results/sample_v2.json

```diff
@@ -1280,9 +1280,11 @@
             "observedSpeedIndex": 4417,
             "observedSpeedIndexTs": 185607736763
           },
-          {
-            "lcpInvalidated": false
-          }
+          [
```

Comment:
oh wait sorry! can we keep this just an object *inside* the items array?

Reply:
`debugInfo` on the timing summary artifact could probably just be an object, it was only an array here because it had to be in items :)